packages/components/src/components/user/CreateUsersModal.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react'
+import { Checkbox, FormControl, Modal } from "react-bootstrap";
+import { Security } from "@labkey/api";
+import { WizardNavButtons } from "../buttons/WizardNavButtons";
+import { Alert } from "../base/Alert";
+import { SelectInput } from "../forms/input/SelectInput";
+
+interface Props {
+    onCancel: () => void
+    onComplete: (response: any, role: string) => void
+    show: boolean
+
+    // optional array of role options, objects with id and label values
```

Comment:
Is it worth making an interface for these objects?

Reply:
The objects just need to contain an id and label, so an interface would be good but might not be worth it here. I'll update the comment to give an example of the objects.